content/blog/2016-11-16-interview-with-canonical.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Interview with Canonical
+created_at: 2016-11-16
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Canonical talks
+about how they are transitioning to Prometheus.*
+
+## Can you tell us about yourself and what Canonical does?
+
+Canonical is probably best known as the company that sponsors Ubuntu Linux.  We
+also produce or contribute to a number of other open-source projects including
+MAAS, Juju, and OpenStack, and provide commercial support for these products.
+Ubuntu powers the majority of OpenStack deployments, with 55% of production
+clouds and [58% of large cloud deployments](
+https://www.openstack.org/assets/survey/April-2016-User-Survey-Report.pdf).
+
+My group, BootStack, is our fully managed private cloud service.  We build and
+operate OpenStack clouds for Canonical customers.
+
+## What was your pre-Prometheus monitoring experience?
+
+We’d used a combination of [Nagios](https://www.nagios.org/),
+[Graphite](https://graphite.readthedocs.io/en/latest/)/[statsd](https://github.com/etsy/statsd),
+and in-house [Django](https://github.com/etsy/statsd) apps.  These did not
```

Comment:
Django links also to statsd. I'd say either change the link target or remove the link.